web/public/mgmt/config.yml

```diff
@@ -1784,22 +1784,71 @@ collections:
     editor:
       preview: true
     fields:
-      - { label: "Content", name: "body", widget: "markdown" }
-      - { label: "Url Slug", name: "urlSlug", widget: "nospace" }
       - { label: "Filename", name: "filename", widget: "nospace", required: false }
-      - { label: "Name", name: "name", widget: "string" }
-      - { label: "Id", name: "id", widget: "nospace" }
-      - { label: "Call To Action Link", name: "callToActionLink", widget: "string", required: false }
-      - { label: "Call To Action Text", name: "callToActionText", widget: "string", required: false }
-      - { label: "Industry", name: "industryId", widget: "string", required: false }
-      - { label: "Issuing Agency", name: "issuingAgency", widget: "string", required: false }
-      - { label: "Issuing Division", name: "issuingDivision", widget: "string", required: false }
-      - { label: "Division Phone", name: "divisionPhone", widget: "string", required: false }
+      - { label: "Url Slug", name: "urlSlug", widget: "nospace" }
+      - { label: "Title (Navigator)", name: "name", widget: "string" }
+      - { label: "Title (Webflow)", name: "webflowName", widget: "string" }
```

Comment:
this is not correct because in these files, the field name from webflow is just called `name`, it's not called `webflowName`

To be honest it probably *should* be called `webflowName` but right now it's not. So you could find-and-replace all of those, I suppose, which would probably make our lives easier in the future when we merge these. Or change the labels here, which will be accurate for now but we'll have to fix things later when we merge. But right now it's incorrect and the webflow name will show in the wrong field